build/sections/founder.html

```diff
@@ -0,0 +1,44 @@
+<div class="bg-[url('../Images/BG_Founder.svg')] bg-no-repeat bg-cover bg-center">
+    <div class="container pt-36">
+        <div class="flex flex-col w-full justify-center md:justify-start text-white font-raleway md:w-[80%] py-5">
+            <h2 class="font-bold text-2xl md:text-4xl">Die <span class="text-primary-green">Founder</span> und ihre Vision von Ojemba</h2>
+            <div class="py-8 text-base font-medium">
+                <p class="pb-3">
+                    Wir werden auf dem afrikanischen Kontinent Wohlstand schaffen, 
+                    indem wir respektvolle gegenseitige Geschäftsbeziehungen 
+                    zwischen dem Norden und dem Süden des Planeten aufbauen. 
+                </p>
+                <p>
+                    Dies wollen wir erreichen, indem wir wir IT-Talente aus dem 
+                    Süden mit IT-Möglichkeiten aus dem Norden zusammenbringen, 
+                    indem wir junge afrikanische Software-Ingenieure ausbilden 
+                    und ihnen anspruchsvolle Jobs für Unternehmen aus dem Norden 
+                    und Süden anbieten.
+                </p>
+            </div>
```

Comment:
Try to use the same font size and font family as in Figma.

Reply:
- When I use the same font size it becomes bigger and even looks bad since the screen size of the Figma is bigger than the one on my laptop. 
> The figma design has a `width` of `
1992px`  and my screen is less than that. Am kindly requesting you bear me on that. Kindly let me know if the texts look similar to the ones on the design.

> For the font family let me work on it now.